src/aselo-webchat.ts

```diff
@@ -0,0 +1,207 @@
+import * as FlexWebChat from '@twilio/flex-webchat-ui';
+import { Channel } from 'twilio-chat/lib/channel';
+import { getUserIp } from './ip-tracker';
+
+getUserIp();
+
+type Translations = {
+  [language: string]: {
+    [key: string]: string;
+  };
+};
+
+const translations: Translations = {
+  'en-US': {
+    WelcomeMessage: "Welcome to Aselo!",
+    MessageCanvasTrayContent: "",
+    MessageInputDisabledReasonHold: "Please hold for a counselor.",
+    AutoFirstMessage: "Incoming webchat contact",
+  },
+  'es': {
+    EntryPointTagline: "Chatea con nosotros",
+    MessageCanvasTrayButton: "EMPEZAR NUEVO CHAT",
+    InvalidPreEngagementMessage: "Los formularios previos al compromiso no se han establecido y son necesarios para iniciar el chat web. Por favor configúrelos ahora en la configuración.",
+    InvalidPreEngagementButton: "Aprende más",
+    InputPlaceHolder: "Escribe un mensaje",
+    TypingIndicator: "{0} está escribiendo ... ",
+    Read: "Visto",
+    MessageSendingDisabled: "El envío de mensajes ha sido desactivado",
+    Today: "HOY",
+    Yesterday: "AYER",
+    Save: "GUARDAR",
+    Reset: "RESETEAR",
+    MessageCharacterCountStatus: "{{currentCharCount}} / {{maxCharCount}}",
+    SendMessageTooltip: "Enviar Mensaje",
+    FieldValidationRequiredField: "Campo requerido",
+    FieldValidationInvalidEmail: "Por favor provea una dirección válida de email",
+
+    PreEngagementDescription: "Comencemos",
+
+    WelcomeMessage: "¡Bienvenido a Aselo!",
+    MessageCanvasTrayContent: "",
+  },
+  'dk': {
+    MessageCanvasTrayContent: "",
+  }
+}
+
+const defaultLanguage = 'en-US';
+const initialLanguage = defaultLanguage;
+
+const appConfig = {
```

Comment:
We could have this in separate files as well, and that way the same code can be used for all the helplines (unless there are specific customization).